README.md

```diff
@@ -324,21 +324,52 @@ specification.
 
 To provide the Olm library in a browser application:
 
- * download the transpiled libolm (from https://packages.matrix.org/npm/olm/).
+ * download the transpiled libolm (from https://gitlab.matrix.org/matrix-org/olm/-/releases).
  * load ``olm.js`` as a ``<script>`` *before* ``browser-matrix.js``.
 
 To provide the Olm library in a node.js application:
 
- * ``yarn add https://packages.matrix.org/npm/olm/olm-3.1.4.tgz``
-   (replace the URL with the latest version you want to use from
-    https://packages.matrix.org/npm/olm/)
- * ``global.Olm = require('olm');`` *before* loading ``matrix-js-sdk``.
+ * ``yarn add @matrix-org/olm --registry=https://gitlab.matrix.org/api/v4/packages/npm/packages/npm/``
+ * ``global.Olm = require('@matrix-org/olm');`` *before* loading ``matrix-js-sdk``.
 
-If you want to package Olm as dependency for your node.js application, you can
-use ``yarn add https://packages.matrix.org/npm/olm/olm-3.1.4.tgz``. If your
-application also works without e2e crypto enabled, add ``--optional`` to mark it
+If your application also works without e2e crypto enabled, add ``--optional`` to mark it
 as an optional dependency.
 
+Besides Olm, you need to configure some options when `createClient()` . These three options are required for enabling e2e crypto: `sessionStore` , `cryptoStore` and `deviceId` .
```

Comment:
Not really sure what's going on with the punctuation here
```suggestion
Besides Olm, you need to configure some options when `createClient()`. These three options are required for enabling e2e crypto: `sessionStore` , `cryptoStore` and `deviceId`.
```

Reply:
this could also be put into the example later on as a comment. For example:

```javascript
const matrixClient = sdk.createClient({
    // Example construction of the client
    baseUrl: "http://localhost:8008",
    accessToken: myAccessToken,
    sessionStore: new sdk.WebStorageSessionStore(localStorage),

    // Values required for crypto to work
    userId: myUserId,
    deviceId: myDeviceId,
    cryptoStore: new sdk.MemoryCryptoStore(),
});
```